boards/arm/lpcxpresso54114/pre_dt_board.cmake

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2019, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Suppress "simple_bus_reg" on LPC boards as all GPIO ports use the same register.
+list(APPEND EXTRA_DTC_FLAGS "-Wno-simple_bus_reg")
```

Comment:
Can we do this in the SoC cmake instead of the board one?

Reply:
We can not unfortunately. The SoC CMake is processed after DT.

Reply:
@galak I've tried it earlier but as @SebastianBoe mentioned SoC is processed after DT.
Also I think having it later on helps us have a finer granularity.
Fixing pylint next.